steps/draft-appeals/content.en.json

```diff
@@ -22,7 +22,19 @@
     "Employment and Support Allowance (ESA)": "Employment and Support Allowance (ESA)",
     "Personal Independence Payment (PIP)": "Personal Independence Payment (PIP)",
     "Universal Credit (UC)": "Universal Credit (UC)",
+    "Attendance Allowance": "Attendance Allowance",
+    "Bereavement Benefit": "Bereavement Benefit",
+    "Bereavement Support Payment Scheme": "Bereavement Support Payment Scheme",
     "Carer's Allowance": "Carers Allowance",
+    "Disability Living Allowance (DLA)": "Disability Living Allowance (DLA)",
+    "Income Support": "Income Support",
+    "Industrial Injuries Disablement Benefit": "Industrial Injuries Disablement Benefit",
+    "Industrial Death Benefit": "Industrial Death Benefit",
+    "Job Seekers Allowance (JSA)": "Job Seekers Allowance (JSA)",
+    "Maternity Allowance": "Maternity Allowance",
+    "Social Fund": "Social Fund",
+    "Pension Credits": "Pension Credits",
+    "Retirement Pension": "Retirement Pension",
     "'Benefit Not Set": "Benefit Not Set"
```

Comment:
There is a extra ' here. It's not done in this pr though. Also welsh content value is different for this key.

Reply:
This is existing "'Benefit Not Set" one, no change in that line